src/Microsoft.AspNetCore.Razor.Language/RazorDiagnosticFactory.cs

```diff
@@ -55,6 +55,16 @@ public static RazorDiagnostic CreateParsing_DuplicateDirective(string directive,
             return RazorDiagnostic.Create(Parsing_DuplicateDirective, location, directive);
         }
 
+        internal static readonly RazorDiagnosticDescriptor Parsing_SectionsCannotBeNested =
+            new RazorDiagnosticDescriptor(
+                $"{DiagnosticPrefix}2002",
+                () => LegacyResources.ParseError_Sections_Cannot_Be_Nested,
```

Comment:
Should this move from `LegacyResources` to `Resources`?

Reply:
Yes

Reply:
I'll keep then in LegacyResources for now. I'll move them all together in a separate PR once we get rid of the legacy diagnostic.

Reply:
yes, these should eventually move. The goal is to nuke `LegacyResources`